startup.py

```diff
@@ -156,6 +164,40 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
     ##########################################################################################
     # private methods
 
+    def __setup_pythonpath(self, alias_code_name, alias_exec_path):
+        """
+        Setup the PYTHONPATH to access Alias API according to Alias and Python versions
+        :return:
+        """
+
+        alias_code_name = alias_code_name or self.FALLBACK_CODE_NAME
+        alias_release_version = self._get_release_version(
+            alias_exec_path, alias_code_name
+        )
+        python_major_version = self._get_python_version()
+
+        api_folder_name = None
+        for api_folder in self.ALIAS_API:
+            min_version = self.ALIAS_API[api_folder].get("min_version")
+            max_version = self.ALIAS_API[api_folder].get("max_version")
+            if min_version and alias_release_version < min_version:
+                continue
+            if max_version and alias_release_version > max_version:
+                continue
+            api_folder_name = api_folder
+
+        if not api_folder_name:
+            return
+
+        python_path = os.path.join(
+            self.disk_location,
+            "api",
+            "python{major_version}".format(major_version=python_major_version),
+            api_folder_name,
+        )
+
+        sgtk.util.append_path_to_env_var("PYTHONPATH", python_path)
```

Comment:
Have you tried testing with prepend_path_to_env_var here? I noted that Alias will find v3.7.4 from my local Python install ahead of the SG Desktop v3.7.6 in my tests yesterday.